src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportControl.java

```diff
@@ -31,7 +31,7 @@
  * be the root node. Each instance may have zero or more children.
  *
  */
-class ImportControl {
+/* default */ class ImportControl {
```

Comment:
public

if smb create a class in his application, there is not harm for us, it is his business. As sources are open why we should limit them to create classes ?